sklearn/tests/test_multioutput.py

```diff
@@ -648,3 +653,116 @@ def test_classifier_chain_tuple_invalid_order():
 
     with pytest.raises(ValueError, match='invalid order'):
         chain.fit(X, y)
+
+
+def test_classifier_chain_verbose(capfd):
```

Comment:
Could you parametrize in the same spirit than there: https://github.com/scikit-learn/scikit-learn/pull/16069/files#diff-a9256841e2ca6477c25abca76d3b5d785b69160e055765031b3e9cd8ae3b7b51R517-R539